libmscore/chordlist.cpp

```diff
@@ -607,9 +616,9 @@ bool ParsedChord::parse(const QString& s, const ChordList* cl, bool syntaxOnly,
             addToken(QString(s[i++]),ChordTokenClass::EXTENSION);
 #endif
       lastLeadingToken = _tokenList.size();
-      // get extension - up to first non-digit
+      // get extension - up to first non-digit other than comma or slash
```

Comment:
Why? Was there a mistake?

Reply:
More like a refinement.  Currently these parse as separate tokens, which required special case handling in the XML files for the chord C6/9 and C6,9.  But then we would have needed to also special-case Cm6/9, Cmi6,9, C-6/9, etc.  This change allows the parser to see 6/9 and 6,9 as a single token, which allows all those variations to work without special casing in the XML file.